deploy-service/teletraanservice/src/test/java/com/pinterest/teletraan/worker/AutoPromoteBuildTest.java

```diff
@@ -54,72 +75,37 @@ public void setUp() throws Exception {
         context.setBuildTagsManager(buildTagsManager);
         context.setBuildDAO(buildDAO);
         context.setTagDAO(tagDAO);
-        first.setBuild_id("0000001");
-        second.setBuild_id("0000002");
         environBean.setEnv_id("test1");
 
+        t8AMBuildBean = new BuildBean();
+        t8AMBuildBean.setBuild_id("8AMBuildId");
+        t8AMBuildBean.setBuild_name("8AMBuildName");
+        t8AMBuildBean.setCommit_date(t9AM.minusHours(2).getMillis());
+        t8AMBuildBean.setPublish_date(t9AM.minusHours(1).getMillis());
+        t8AMBuildBean.setScm_commit("fghij");
+
+        t9AMBuildBean = new BuildBean();
+        t9AMBuildBean.setBuild_id("9AMBuildId");
+        t9AMBuildBean.setBuild_name("9AMBuildName");
+        t9AMBuildBean.setCommit_date(t9AM.minusHours(1).getMillis());
+        t9AMBuildBean.setPublish_date(t9AM.getMillis());
+        t9AMBuildBean.setScm_commit("abcde");
+
+        t10AMPromoteBean = new PromoteBean();
+        t10AMPromoteBean.setEnv_id(environBean.getEnv_id());
+        t10AMPromoteBean.setSchedule(CronTenAMPerDay);
+        t10AMPromoteBean.setLast_update(0L);
+        t10AMPromoteBean.setDelay(0);
     }
 
-
-    @Test
-    public void testGetScheduledCheckDueTime() throws Exception {
```

Comment:
why removing this test?

Reply:
This is removed along with the no-longer-referenced function `GetScheduledCheckDueTime`.